_delphi_utils_python/delphi_utils/flash_eval/run.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the tool.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m delphi_utils.flash_eval`.
+"""
+from datetime import date
+import pandas as pd
+from .. import read_params
+from .eval_day import flash_eval
+from ..validator.datafetcher import read_filenames, load_csv
+
+def run_module():
+    """Run the FlaSH module.
+
+    The parameters dictionary must include the signals and signals.
+    We are only considering lag-1 data.
+    """
+    params = read_params()
+    signals = params["flash"]["signals"]
+    for signal in signals:
+        export_files = read_filenames(params["common"]["export_dir"])
+        days = {}
+        for (x, _) in export_files:
+            #Concat the data from recent files at nation, state, and county resolution per day.
```

Comment:
question (non-blocking): Why are we combining across geo types?

Don't we want to process them separately later? If so, it would be more convenient (less work) to keep them separate and use two keys, date and geo_value, in our signal-df dictionary below.

Reply:
Ah, so for FlaSH, it currently needs to pool across streams with the same geography (combine the streams for all the counties in Texas + the state of Texas). So I process them all together!